packages/cubejs-teradata-jdbc-driver/package.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Please remove the terajdbc4.jar file from the PR.

Reply:
done